### PR TITLE
Added NumbersClient class

### LINKED ISSUE #12: Bandwidth Client class
Need some type of BandwidthClient class that stores authentication credentials

### PR DESCRIPTION
**Pull request by jmulford-bandwidth**
closes #12

### CODE REVIEW

File: src/main/java/com/bandwidth/sdk/numbers/NumbersClient.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    public NumbersClient(String account, String username, String password) {
+        this.account = account;
+
+        AsyncHttpClientConfig httpClientConfig = new DefaultAsyncHttpClientConfig.Builder()
+                .setRealm(new Realm.Builder(username, password)
+                        .setUsePreemptiveAuth(true)
+                        .setScheme(Realm.AuthScheme.BASIC))
+                .build();
+
+        httpClient = asyncHttpClient(httpClientConfig);
```
Comment (fuchsnj) [inline src/main/java/com/bandwidth/sdk/numbers/NumbersClient.java]: you need to import this function
`import static org.asynchttpclient.Dsl.asyncHttpClient;`

Comment (jmulford-bandwidth) [reply src/main/java/com/bandwidth/sdk/numbers/NumbersClient.java]: done

**Comment by jmulford-bandwidth** [comment]
@NathanNorman build.gradle is in master

**Comment by jmulford-bandwidth** [comment]
Already done

**Pull request closed without merging**